wallet/src/__test__/client/index.test.ts

```diff
@@ -0,0 +1,63 @@
+import { Sequelize } from "sequelize";
+import WalletClientGenerator, {
+  CreateWalletRequest,
+  WalletClient,
+} from "../../client";
+import database from "../../database";
+import { WalletDAO } from "../../database/dao/WalletDAO";
+import RapydClient from "../../lib/rapyd/RapydClient";
+import { WalletObjectResponse } from "../../lib/rapyd/types";
+import getRandomUsername from "../util/getRandomUsername";
+
+let driver: Sequelize,
+  dao: WalletDAO,
+  walletClient: WalletClient,
+  rapydClient: RapydClient;
+
+describe("controller", () => {
+  beforeAll(async () => {
+    driver = await database.connect({ force: true });
+    dao = new WalletDAO(driver);
+    walletClient = new WalletClientGenerator(
+      `${process.env.IP_ADDRESS}:${process.env.HTTP_PORT}`
+    ).create();
+    rapydClient = new RapydClient();
+  });
+
+  test("success: creates a real Rapyd wallet per user and stores it in the database and in Rapyd", async () => {
```

Comment:
@Ktoxcon I've begun testing the WalletClient, it's not fully working yet, but I'm getting close. 

I'm getting this error on the `GET wallet` endpoint: 

<img width="1440" alt="Screen Shot 2021-06-01 at 18 53 06" src="https://user-images.githubusercontent.com/4053518/120404011-a4f80280-c302-11eb-83fb-635e4ce47da1.png">